Ginger/Ginger/Activities/ActivitiesGroupSelectionPage.xaml.cs

```diff
@@ -101,5 +109,23 @@ private void SelectBtn_Click(object sender, RoutedEventArgs e)
             }
             mPageGenericWin.Close();
         }
+
+        private void xLinkedInstance_Checked(object sender, RoutedEventArgs e)
+        {
+            if (xNoteLable == null)
+            {
+                return;
+            }
+            xNoteLable.Content = "Note- * If we update any instance of this type, all the flows having this instance will be updated.";
+        }
+
+        private void xRegularInstance_Checked(object sender, RoutedEventArgs e)
+        {
+            if (xNoteLable == null)
+            {
+                return;
+            }
+            xNoteLable.Content = "Note- * If we update any instance of this type, we need to manually updated all the flows with this instance.";
```

Comment:
change line to below 
Note - Any updation to instance of this type, needs manual updation of all the flows with this instance.

Reply:
Done